plugins/kadalu.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kadalu
+spec:
+  version: "v0.8.7"
+  homepage: https://github.com/kadalu/kadalu
```

Comment:
do you import auth plugins in your program? 
https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/#:~:text=in%20your%20plugins.-,Import%20authentication%20plugins%20(Go),-By%20default%2C%20plugins

Reply:
Nope

Reply:
@vatsa287 we highly recommend importing that package as it severely excludes your user base from clusters running on managed-Kubernetes services from cloud providers.